client/network/src/lib.rs

```diff
@@ -452,3 +461,51 @@ async fn wait_to_announce<Block: BlockT>(
 		);
 	}
 }
+
+/// A [`BlockAnnounceValidator`] which accepts all block announcements, as it assumes
+/// sybil resistance is handled elsewhere.
+#[derive(Debug, Clone)]
+pub struct AssumeSybilResistance(bool);
+
+impl AssumeSybilResistance {
+	/// Instantiate this block announcement validator while permissively allowing (but ignoring)
```

Comment:
The "(but ignoring)" from this line seems like it clashes a bit with "shouldn't be ignored" from line 475. 

Perhaps the difference is at which step we ignore certain announcements?

Reply:
Yeah, that's right. Better phrased below would be "shouldn't be rejected"